astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1581,6 +1581,9 @@ def blockstart_tolineno(self):
         """
         return self.args.tolineno
 
+    def implicit_parameters(self):
```

Comment:
```suggestion
    def implicit_parameters(self):
```

Could you add typing with `Literal` here?